src/components/Signup/Signup.test.js

```diff
@@ -35,5 +35,69 @@ describe('Signup', () => {
 
     // Assert
     expect(container.querySelectorAll('.error')).not.toHaveLength(0);
+  });  
+
+  it('should show Argentina below Argelia', () => {
+    // Arrange
+    const { container } = render(
+      <AppServicesProvider>
+        <DopplerIntlProvider locale='es'>
+          <Router>
+            <Signup />
+          </Router>
+        </DopplerIntlProvider>
+      </AppServicesProvider>,
+    );
+     
+    // Assert
+    expect(container.querySelector('#iti-item-dz').nextElementSibling.id).toBe('iti-item-ar');
+  });  
+
+  it('should show Territorio Palstino, Ocupado below Territorio Britanico del Oceano Indico', () => {
```

Comment:
```suggestion
  it('should show Territorio Palestino, Ocupado below Territorio Britanico del Oceano Indico', () => {
```

Reply:
What do you think about also indicate the language in the test name?

Reply:
I'm agree. Thanks!